3761/1/initial_list_of_gems.rb

```diff
@@ -0,0 +1,23 @@
+# read file with gems and return initial list of gems
+class InitialListOfGems
```

Comment:
Bad class name. https://medium.com/selleo/a-subjective-guide-to-naming-stuff-in-ruby-on-rails-classes-b44928b6c49a